src/Microsoft.AspNetCore.Antiforgery/Internal/DefaultAntiforgery.cs

```diff
@@ -102,11 +106,14 @@ public AntiforgeryTokenSet GetTokens(HttpContext httpContext)
                 throw new ArgumentNullException(nameof(httpContext));
             }
 
+            _logger.ValidatingAntiforgeryToken();
+
             CheckSSLConfig(httpContext);
```

Comment:
This method will `throw` if it's an HTTP request and Antiforgery is configured to require HTTPS. Should that event be logged?